Adyen/Assets/pl-PL.lproj/Localizable.strings

```diff
@@ -86,6 +86,7 @@
 "adyen.billingAddressSection.title" = "Adres rozliczeniowy";
 "adyen.billingAddress.placeholder" = "Twój adres rozliczeniowy";
 "adyen.deliveryAddressSection.title" = "Adres dostawy";
+"adyen.deliveryAddress.placeholder" = "Adres dostawy";
```

Comment:
```suggestion
"adyen.deliveryAddress.placeholder" = "Twój adres adres dostawy";
```

Reply:
Thanks for the suggestion - will fix it in a subsequent PR